### ISSUE TITLE
Serializing EmfServer

### ISSUE THREAD

**Issue by sdybiec** [qgears/opensource-utils#4]
I've made some progress attaching emfcollab to my EMF-generated editor. The client and server are communicating, and the EmfCollabEditorExtension is wired into the editor. Here's my problem: EmfCollabEditorExtension.loadModel() makes the following call:

EmfServer server = multiServer.getServerForResource(sessionKey, credentials, resourceName);

This requires that the EmfServer instance be serialized, but it is does now implement 'Serializable'. Did you use a 'CoolRMIReplaceEntry' to deal with this somehow?

Thanks.

**Comment by rizsi** [comment]
Yes, you are right. Brilliant analysis of a complex topic! The user of the lib has to configure the EmfServer class to be a remoting stub of the IEmfServer interface when the CoolRMI service is created. So all calls to this object on the client side are remoted to the server automatically. We created this library several years ago but we never got to the point to publicate the usage example. We are not testing it currently. If you intend to use it then it may be necessary to dust it a little.

**Comment by sdybiec** [comment]
Thanks. Your comment was helpful in getting it working.  

The sample EMF 'Library' sample model I'm using for testing EMFCollab contains FeatureMaps and multiplicity-many attributes, and it seems those are not yet supported by EMFCollab. Perhaps worth a contribution if I figure out how to get these working. Thanks again for your help.

**Comment by rizsi** [comment]
Yes, we have only implemented the minimal features that we needed. Your welcome, feel free to ask!
